unit_tests/src/test_rc_data_obj.cpp

```diff
@@ -548,3 +664,13 @@ auto create_empty_replica(const fs::path& _path) -> void
     REQUIRE(GOOD_REPLICA == replica::replica_status(comm, _path, 0));
 } // create_empty_replica
 
+auto create_local_file_40mb(const std::string_view _path, char _ch) -> void
```

Comment:
This could be made a little more generic and moved into the unit test utility header.